setup.py

```diff
@@ -73,6 +73,7 @@
     "sanic~=19.9",
     "sanic-cors==0.9.9.post1",
     "sanic-jwt~=1.3",
+    "multidict==4.6.1",
```

Comment:
@amn41 Can you please also add a comment here (maybe even with a link to the sanic issue). Same for requirements.txt